packages/transition/src/slide-fade.tsx

```diff
@@ -57,12 +57,12 @@ export interface SlideFadeProps extends HTMLMotionProps<"div"> {
    * The offset on the horizontal or `x` axis
    * @default 0
    */
-  offsetX?: number
+  offsetX?: string
   /**
    * The offset on the vertical or `y` axis
    * @default 8
    */
-  offsetY?: number
+  offsetY?: string
```

Comment:
I think for now we should allow them to be `string | number`. It can actually handle far more than that but I need to figure out how to access that specific type within `framer-motion`, so expanding this a bit is a good start.

Reply:
Ok, in my free time, I will figure out how to expand these props, but, looking inside the `framer-motion` function that we use, they have something more inside that we need to look, but I will expand to number and string.

But based on the docs, this component only accept string int both offsets.

Reply:
@jvmazagao The types in the docs are generated from the types in the codebase, so it'll match whatever you put here 😉

Reply:
Fixed